hcloud/actions/client.py

```diff
@@ -36,6 +50,18 @@ def get_list(self,
                  per_page=None,  # type: Optional[int]
                  ):
         # type: (...) -> PageResults[List[BoundAction]]
+        """Get a list of actions from this account
+
+        :param status: List[str] (optional)
+               Response will have only actions with specified statuses. See `our documentation <https://docs.hetzner.cloud/#actions-list-all-actions>`_  for all available values.
```

Comment:
Try to be more specific. Put main information in the doc string and refer to API docs only for details. 
Example: user can see all available values for "status" param right away in the docstring.

Same in other parts of documentation.

Reply:
What if the strings would be changed in the future? We would need to release a new hcloud-python version then, instead of just using our API Documentation which should be the single point of truth.

Reply:
Changing the string is a "breaking change". Not only our API will fail, but all users application as well.

Reply:
Is already replaced on all parts.